diff --git a/vanetza/asn1/tests/CMakeLists.txt b/vanetza/asn1/tests/CMakeLists.txt
index c003988..6ff37e5 100644
--- a/vanetza/asn1/tests/CMakeLists.txt
+++ b/vanetza/asn1/tests/CMakeLists.txt
@@ -7,6 +7,7 @@ add_gtest(ItsAsn1 SOURCES its.cpp)
 add_gtest(ItsAsn1Rel2 SOURCES its_r2.cpp LINK_LIBRARIES asn1_its_r2)
 add_gtest(PacketVisitor SOURCES packet_visitor.cpp LINK_LIBRARIES net)
 add_gtest(SecurityAsn1 SOURCES security.cpp LINK_LIBRARIES asn1_security)
+add_gtest(SpatemMapemAsn1 SOURCES spatem_mapem.cpp)
 
 add_library(asn1_smoke_test STATIC smoke.cpp)
 target_link_libraries(asn1_smoke_test PRIVATE asn1)
