diff --git a/vanetza/facilities/CMakeLists.txt b/vanetza/facilities/CMakeLists.txt
index caffa18..ae1113c 100644
--- a/vanetza/facilities/CMakeLists.txt
+++ b/vanetza/facilities/CMakeLists.txt
@@ -1,7 +1,10 @@
 set(CXX_SOURCES
     cam_functions.cpp
+    map_functions.cpp
     path_history.cpp
     path_point.cpp
+    priority_functions.cpp
+    spat_functions.cpp
 )
 
 add_vanetza_component(facilities ${CXX_SOURCES})
diff --git a/vanetza/facilities/tests/CMakeLists.txt b/vanetza/facilities/tests/CMakeLists.txt
index 77239fd..72ef165 100644
--- a/vanetza/facilities/tests/CMakeLists.txt
+++ b/vanetza/facilities/tests/CMakeLists.txt
@@ -2,5 +2,8 @@ include(UseGTest)
 configure_gtest_directory(LINK_LIBRARIES facilities)
 
 add_gtest(CamFunctions cam_functions.cpp LINK_LIBRARIES asn1_its_r2)
+add_gtest(MapFunctions map_functions.cpp)
 add_gtest(PathHistory path_history.cpp)
 add_gtest(PathPoint path_point.cpp)
+add_gtest(PriorityFunctions priority_functions.cpp)
+add_gtest(SpatFunctions spat_functions.cpp)
